main/common/constants.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const supportedVideoExtensions = ['mp4', 'mov', 'm4v'];
-const defaultInputDevice = {id: 'SYSTEM_DEFAULT', name: 'System Default'};
 
 module.exports = {
   supportedVideoExtensions,
-  defaultInputDevice
+  defaultInputDevice: {id: 'SYSTEM_DEFAULT', name: 'System Default'},
+  builtInMicrophoneId: 'BuiltInMicrophoneDevice'
```

Comment:
I think `macos-audio-devices` should expose this. And maybe also a `isBuiltIn` property on the device info.

Reply:
Well, I'm not sure if that is actually right. I've just noticed it in my machines. I don't know if there's a way to know for sure what device is built in